### PR TITLE
use a single source of truth, redpanda.yaml

### PR DESCRIPTION
**Pull request by joejulian**

### CODE REVIEW

File: redpanda/templates/statefulset.yaml [hunk 1]
```diff
@@ -198,19 +198,6 @@ spec:
...
-            - --rpc-addr=$(POD_IP):{{ .Values.listeners.rpc.port }}
-            - --schema-registry-addr=internal://$(POD_IP):{{ .Values.listeners.schemaRegistry.port }},
-{{- range $name, $listener := .Values.listeners.schemaRegistry.external -}}
-                {{ $name }}://$(POD_IP):{{ $listener.port }},
-{{- end }}
-            - --pandaproxy-addr=internal://$(POD_IP):{{ .Values.listeners.http.port }},
-{{- range $name, $listener := .Values.listeners.http.endpoints -}}
-                {{ $name }}://$(POD_IP):{{ $listener.port }},
-  {{- end }}
             - --advertise-kafka-addr=internal://{{ $advertiseAddress }}:{{ .Values.listeners.kafka.port }},
```
[the remark this answers is not in the document]
Comment (esteban) [reply redpanda/templates/statefulset.yaml:201]: We will still pass `--advertise-kafka-addr=internal` even after removing the other arguments.

[the remark this answers is not in the document]
Comment (joejulian) [reply redpanda/templates/statefulset.yaml:201]: That's the advertise address, which is pod specific.

[the remark this answers is not in the document]
Comment (joejulian) [reply redpanda/templates/statefulset.yaml:201]: As far as I can tell, this seems to need to stay here.

[the remark this answers is not in the document]
Comment (esteban) [reply redpanda/templates/statefulset.yaml:201]: Yeah, I think it should be fine. Thank.

**Review by esteban — APPROVED** [review]

**Pull request merged**